### PR TITLE
Implement function 'CreateApplicationStep'

### PR DESCRIPTION
**Pull request by jasonburkert-ccpo**
This DRAFT PR is a work in progress.

TODO
- [ ] implement functionality
- [ ] implement input validation
- [ ] implement unit tests
- [ ] test in AWS

Ticket: AT-6467

### CODE REVIEW

File: packages/api/portfolioDrafts/application/createApplicationStep.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  if (!portfolioDraftId) {
+    return NO_SUCH_PORTFOLIO_DRAFT;
+  }
+
+  if (!isValidJson(event.body)) {
+    return REQUEST_BODY_INVALID;
+  }
+  const requestBody = JSON.parse(event.body);
+
+  // remove this, and function?
```
Comment (jasonburkert-ccpo) [inline packages/api/portfolioDrafts/application/createApplicationStep.ts:31]: Remove in light of Zach's schema validation work?  Or leave for now?

File: packages/api/portfolioDrafts/application/createApplicationStep.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  if (!isApplicationStep(requestBody)) {
+    return REQUEST_BODY_INVALID;
+  }
```
Comment (jasonburkert-ccpo) [inline packages/api/portfolioDrafts/application/createApplicationStep.ts:34]: Remove in light of Zach's schema validation work?  Or leave for now?

Comment (zachclark-ccpo) [reply packages/api/portfolioDrafts/application/createApplicationStep.ts:34]: I would leave it for now.

**Comment by jasonburkert-ccpo** [comment]
Closing due to substitute branch `feature/AT-6467-fixed`

**Pull request closed without merging**